components/automate-cli/cmd/chef-automate/gather-logs.go

```diff
@@ -398,8 +398,7 @@ func runGatherLogsLocalCmd(outfileOverride string, logLines uint64) error {
 	// TODO: This might not always be the right choice, we should look into
 	// assigning this dynamically or allow a config flag. Right now /tmp
 	// works on our target distros and this is just an emergency fallback
-	stagingDir := "/tmp"
-	archiveRoot, err := createTempDir(stagingDir, "log-gathering")
```

Comment:
we can remove this as well 
gather-logs command is writing the logs into the /tmp
that automatically cleanup